packages/privacy-toolset/README.md

```diff
@@ -0,0 +1,37 @@
+# Components
+
+A library of tools created as React components designed to unify privacy mechanism in Automattic products.
+
+## Installation
+
+Install the toolset.
+
+```bash
+yarn add @automattic/privacy-toolset
+```
+
+## Usage
+
+```jsx
+// import the component you wish to use
+import { CookieBanner } from '@automattic/privacy-toolset';
+
+const Component = () => (
+	<>
+		<CookieBanner content={ ...args } onAccept={ fn } />
```

Comment:
Since the required props are nested, providing an example of the structure here would be nice for those implementing the package. I think the storybook component could serve this purpose, could potentially reference that here.

Reply:
Especially as the CookieBanner component is fully typed, I'm not sure if that's necessarily needed - I used this example mainly to explain how to import components from this library in general. However, I agree that some disclaimers will clarify where to seek more info on param definitions.